documentation/optee_design.md

```diff
@@ -611,68 +431,68 @@ struct optee_header {
         uint32_t paged_size;
 };
 ```
+
 The header is only used by the loader of OP-TEE, not OP-TEE itself. To
-initialize OP-TEE the loader loads the complete binary into memory and
-copies what follows the header and the following ```init_size``` bytes to
-```(init_load_addr_hi << 32 | init_load_addr_lo)```. ```init_mem_usage``` is
-used by the loader to be able to check that there's enough physical memory
-available for OP-TEE to be able to initialize at all. The loader supplies
-the address of the first byte following what wasn't copied in r0/x0 and jumps
-to the load address to start OP-TEE.
+initialize OP-TEE the loader loads the complete binary into memory and copies
+what follows the header and the following `init_size` bytes to
+`(init_load_addr_hi << 32 | init_load_addr_lo)`. `init_mem_usage` is used by the
+loader to be able to check that there's enough physical memory available for
+OP-TEE to be able to initialize at all. The loader supplies the address of the
+first byte following what wasn't copied in r0/x0 and jumps to the load address
```

Comment:
-The loader supplies the address of the first byte following what wasn't copied in r0/x0 and jumps
+The loader supplies in r0/x0 the address of the first byte following what wasn't copied and jumps